ui/pages/settings/security-tab/security-tab.component.js

```diff
@@ -185,7 +185,11 @@ export default class SecurityTab extends PureComponent {
   renderSecurityAlertsToggle() {
     const { t } = this.context;
 
-    const { securityAlertsEnabled } = this.props;
+    const {
+      ///: BEGIN:ONLY_INCLUDE_IF(blockaid)
+      securityAlertsEnabled,
+      ///: END:ONLY_INCLUDE_IF
+    } = this.props;
```

Comment:
this value is called on line 228. We should be able to revert this code fence because we have the code fence around the render function

```
{/* ///: BEGIN:ONLY_INCLUDE_IF(blockaid) */}
   {this.renderSecurityAlertsToggle()}
{/* ///: END:ONLY_INCLUDE_IF */}
```

```suggestion
    const { securityAlertsEnabled } = this.props;
```

Reply:
Updated [here](https://github.com/MetaMask/metamask-extension/pull/23460/commits/258413ba5d97c69a241d5d1653ac9d04f5fd0ca2)